microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -18,12 +18,21 @@ In addition to loading remote content, content can also be loaded locally into W
 
 These approaches are described below.
 
```

Comment:
DR's question 3 from main page of PR:

tbd

Reply:
@david-risney -- I think the table (whether text cells or emojis) effectively complements the details in the h2 sections.

It's possible for you to incorp my PR comments and then the PR could be merged.  Maybe in a later PR if needed we could add a structured, consistent list / h4 section in each h2 section.  

Locally I drafted such an h4 (Limitations of loading local content by navigating to an HTML string) and h6's (eg Origin-based DOM APIs) -- In h2 section "Navigate to an HTML string".  We might want to implement this consistent set of headings in each h2.

Reply:
> It's possible for you to incorp my PR comments and then the PR could be merged. Maybe in a later PR if needed we could add a structured, consistent list / h4 section in each h2 section.

Yes that sounds good to me.

Reply:
We ended up with me creating a PR to add h6s for each row of the new table, in each main h2 section, then merged that PR into this PR.  The present thread can be Resolved.